service/history/interfaces/mutable_state.go

```diff
@@ -212,6 +212,9 @@ type (
 		TaskQueueScheduleToStartTimeout(name string) (*taskqueuepb.TaskQueue, *durationpb.Duration)
 
 		IsWorkflowExecutionRunning() bool
+		IsWorkflowExecutionStatusPaused() bool
```

Comment:
it can also be done with `ms.GetExecutionState().Status == enumspb.WORKFLOW_EXECUTION_STATUS_PAUSED`, new method isn't really necessary.

Reply:
There are some follow-up PRs where I'm using this. They appear inside a if statement with multiple conditionals. So it comes handy. I'll remove it from here and add it in the PR where i'm using it so that it becomes clear.